vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -434,8 +434,9 @@ ${stderr}`));
         }
         else
         {
-            this.context?.eventStream.post(new CommandExecutionEvent(`Executing command ${fullCommandString}
-with options ${JSON.stringify(options)}.`));
+            const { env, ...optionsWithoutEnv } = options;
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            this.context?.eventStream.post(new CommandExecutionEvent(`Executing command ${fullCommandString} with options ${JSON.stringify(options.env !== null && options.env !== undefined ? { env: minimizeEnvironment(env), ...optionsWithoutEnv } : options)}.`));
```

Comment:
tiny nit:
```suggestion
            this.context?.eventStream.post(new CommandExecutionEvent(`Executing command ${fullCommandString} with options ${JSON.stringify(options.env !== null && options.env !== undefined ? { env: minimizeEnvironment(env), ...optionsWithoutEnv } : optionsWithoutEnv)}.`));
```
? May as well not send an extra undefined if present. Wondering if you should do the same if env is empty

Reply:
I'd actually be interested in when the env is undefined or not, but that's a thoughtful suggestion.